### ISSUE TITLE
Beta tag on the weather model and UG for use with the WM GST

### ISSUE THREAD

**Issue by ceceliadid** [ufs-community/ufs-mrweather-app#72]
The GST requires a beta tag on the weather model and weather model UG this week, ideally by 1/29. The current head of the ufs_public_release_branch of the weather model is fine for this tag.

**Comment by ceceliadid** [comment]
Hi @junwang-noaa and @arunchawla-NOAA , is there a current ETA for a tag?
Thanks -

**Comment by junwang-noaa** [comment]
I am waiting for the tag name for fv3 dycore finalized.

**Comment by climbfuji** [comment]
We need to merge the missing documentation updates, then we can create the tag for the dycore.

**Comment by ceceliadid** [comment]
Sounds good, thanks.

**Comment by ligiabernardet** [comment]
If necessary, the missing FV3 dycore documentation can be added later. It
is not critical for the GST, and the HTML is already posted.

**Comment by ceceliadid** [comment]
Thanks @ligiabernardet if that could happen that would be great.
I know everyone is beyond busy, but is there any chance someone could put a tag on he WM as it is so we can start the external testing? 
Looking at what I think is the doc update ticket, it looks like it's going to take a while longer, but maybe there's something I'm not seeing.

**Comment by arunchawla-NOAA** [comment]
As soon as the sub component tags are there we will create the model tag

**Comment by ceceliadid** [comment]
@arunchawla-NOAA  Is there any significant contribution you are waiting on? Just trying to get a sense of the timeline.

**Comment by arunchawla-NOAA** [comment]
FV3 dycore tag was created. We should expect the beta tag to be created by tomorrow. Last set of PRs happening right now

**Comment by ceceliadid** [comment]
Hi @llpcarson is there a ufs-v1.0.0.beta01 tag on the weather model UG? If so, what URL should be used to access it? If not, do you know who might be able to do this? 
Including @climbfuji and @junwang-noaa in case they know anything about it.

**Comment by llpcarson** [comment]
No, not specifically for the UG.  It currently exists in a branch called
"documentation".  I thought we were going to merge this into the release
branch at some point, where it would share the same tagging info.   I don't
think we can tag it with this name, though, since there's already a tag in
this repo for the code release branch.

Would it be worthwhile to have an independent tag at this point?  Or wait
for a merge into the release branch and a shared tag?

I'm don't think I have permissions to create tags in this repo, so may need
assistance from Dom or Jun.

Laurie

**Comment by ceceliadid** [comment]
Hi @llpcarson , yes, to start the GST we asked for a tag on the WM UG. I don't think it matters what the specific name of the tag is, it could be something like ufs-v1.0.0.beta01doc if that is okay with @arunchawla-NOAA and others.
That way the UG will not be changing as people are looking at it.

**Comment by llpcarson** [comment]
OK - I was able to push a tag to the repo, and configured ReadTheDocs:

https://ufs-mr-weather-app.readthedocs.io/projects/ufs-weather-model/en/ufs-v1.0.0.beta01doc/

Laurie

**Comment by ceceliadid** [comment]
That is super awesome, thank you! I am going to modify the WM pages and close this ticket!
